src/swap.c

```diff
@@ -157,6 +157,8 @@ static int swap_init (void) /* {{{ */
 /* #endif defined(VM_SWAPUSAGE) */
 
 #elif HAVE_LIBKVM_GETSWAPINFO
+	char errbuf[1024];
```

Comment:
According to http://www.unix.com/man-page/FreeBSD/3/kvm_openfiles/ the size should be `_POSIX2_LINE_MAX` rather than 1024. Could you please correct this here as well as in the processes plugin ? tcpconns seems to already have this correct.